index.html

```diff
@@ -28,18 +29,30 @@
 
         
         <nav>
-            <img class="nav" src="Images/menu-button.png" alt="">
+            <img class="nav" id="myBtn" src="Images/menu-button.png" alt="">
             <ul class="nav-link">
                 <li><a href="">Portfolio</a></li>
                 <li><a href="">About</a></li>
                 <li><a href="">Contact</a></li>
             </ul>
```

Comment:
- [ ] Kindly 😃 try to stick to the Figma Design as you see here that you use rgba for the background color, so please to use the correct one from the Figma ```#6070FF``` to be like the Figma Design 😊
- [ ] Also please try to make the background color blur like the Figma Design that make it nice for the eye you can use this property to make it ⤵️

```snippet

element{
mix-blend-mode: multiply;
backdrop-filter:blur(5px);
}

```

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊


| Figma design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/225031254-e77bfe3d-0c64-4238-86c1-1155f41e68c6.png)    |    ![image](https://user-images.githubusercontent.com/83360636/225031357-35eeb7a9-0625-4056-8d3c-55e8af2b3c63.png)  |